modules/perception/inference/utils/binary_data.h

```diff
@@ -17,11 +17,10 @@
 
 #include <map>
 #include <string>
-
-#include <boost/shared_ptr.hpp>
 #include <limits>
 
 #include "modules/perception/base/blob.h"
+#include <boost/shared_ptr.hpp>
```

Comment:
Seems this header is not on the right place. What do you think? @storypku

Reply:
> Seems this header is not on the right place. What do you think? @storypku

You are right. Put boost headers above apollo's own headers. 
And,  We prefer `std::shared_ptr` from `<memory>` over `boost::shared_ptr`.